### ISSUE TITLE
Occupation or hobbies related to outdoors

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9994]
**Preferred term label:**
Occupation or hobbies related to outdoors

**Synonyms**
Hunting, gardening, farming, agriculture, horticulture, miners, working in wet fields

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sporotrichosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
HP:6000094
History of outdoor activities
